backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersController.java

```diff
@@ -50,4 +54,27 @@ public UserModel getUserById(
 		return userModelAssembler.toModel(user);
 	}
 
+
+    @PostMapping({ "/{userId}/email-validations"})
+    @Operation(summary = "Verify the status of a confirmation code.", operationId="verify-confirmation-code")
+    public ResponseEntity<?>  verifyConfirmationCodeStatus(
+            @NotBlank(message = "userId must not be null or blank")
+            @Parameter(description = "The ID of the user.", required = true)
+            @PathVariable String userId,
+            @NotBlank(message = "code must not be null or blank")
+            @Parameter(description = "The confirmation code.", required = true)
+            @RequestParam String code
```

Comment:
This should be in the POST body.

Reply:
To verify that the code is valid, use a validator. This allows the endpoint to return 204 when the code is accepted, and 400 when it is rejected.

Reply:
After discussing the validator comment with @CanadianPhilipBarkow, we decided that doing the check in a JSR validator isn't ideal.